frontend/src/components/checklist/ChecklistItemTree.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <table style="width: 100%; border-spacing: 0" :style="getRowStyle()">
+    <tr>
+      <td colspan="2" style="text-wrap: wrap; line-height: 36px; padding-left: 6px">
+        {{ getPositionNumber(checklistItem) }})
+        {{ checklistItem.text }}
+      </td>
+      <td style="max-width: 400px; text-align: right">
+        <div
+          v-for="activity in getActivities(checklistItem)"
+          :key="activity._meta.self"
+          style="display: inline; text-wrap: nowrap; padding-right: 4px"
+        >
+          <ScheduleEntryLinks :activity-promise="activity._meta.load" />
+        </div>
+      </td>
+      <td style="width: 30px">
+        <v-btn
+          v-if="getActivities(checklistItem).length > 0"
+          icon
+          @click="copyToClipboard(checklistItem)"
```

Comment:
i would have expected activities as parameter of copyToClipboard
(that's what you copy, isn't it?)